backend/entityservice/tests/test_project_with_runs_deletes.py

```diff
@@ -3,6 +3,9 @@
 
 
 def test_delete_project_after_creating_run_with_clks(requests, result_type):
-    project, dp1, dp2 = create_project_upload_fake_data(requests, [100, 100], overlap=0.5, result_type=result_type)
+    project, (dp1, dp2) = create_project_upload_fake_data(
+        requests, [100, 100], overlap=0.5, result_type=result_type)
+    project, (dp1, dp2) = create_project_upload_fake_data(
```

Comment:
why do you repeat that line now? It wasn't there before.
Also, why not `project, _ = ...` like in the other tests? `dp1` and `dp2` are unused.

Reply:
The line repeat is unintentional (copy-paste error?). Thanks for pointing it out!

The unused `dp1` and `dp2` were there before. To notice it I'd have to read the whole function and I didn't need to do that to make this simple typing change. But I've fixed it also.